frontend/src/app/forms/steps/pictures-entry/pictures-entry.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input } from '@angular/core';
+import { /*DomSanitizer,*/ SafeResourceUrl } from '@angular/platform-browser';
+import { CameraResultType, CameraSource, Plugins } from '@capacitor/core';
+
+import { FormGroup } from '../../../common/forms/forms';
+import { FormsData } from '../../../model/forms-data.model';
+
+@Component({
+  selector: 'form-pictures-entry',
+  templateUrl: './pictures-entry.component.html',
+  styleUrls: ['./pictures-entry.component.scss'],
+})
+
+/*
+  To save Files as Data.url just change restultType and uncomment everything in this file
+*/
+export class PicturesEntryComponent {
+  constructor(/*private sanitizer: DomSanitizer*/) {}
+  @Input()
+  formsData: FormGroup<FormsData>;
+
+  photo: SafeResourceUrl;
```

Comment:
This property is currently not needed, am I right?

Reply:
This is just if we want to use Data.url but, as we decided to go for base64string i will delete this.